code/modules/power/apc.dm

```diff
@@ -555,13 +558,21 @@
 	else
 		return ..()
 
-/obj/machinery/power/apc/take_damage(damage, damage_type = BRUTE, sound_effect = 1)
-	..()
-	if((malfhack || (stat & BROKEN)) && !opened)
-		if(damage < 10)
-			return
-		health -= damage
-		if(health <= 0)
+/obj/machinery/power/apc/run_obj_armor(damage_amount, damage_type, damage_flag = 0, attack_dir)
+	if(damage_flag == "melee" && damage_amount < 15 && (!(stat & BROKEN) || malfai))
```

Comment:
The apc ignores hit below 15, so an assistant with an extinguisher can't break all apcs for fun. But once the apc is broken open the threshold isn't applied so that dropping the cover to fix the apc isn't tedious.